docs/DeveloperGuide.adoc

```diff
@@ -798,35 +802,126 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |student |delete an event |remove events I no longer need to attend
+
+|`* * *` |student |find an assignment by name |locate details of an assignment without having to go through the entire list
+
+|`* * *` |student |find an event by name |locate details of an event without having to go through the entire list
+
+|`* * *` |student |mark an assignment as complete |tell which assignments are completed and which are not
+
+|`* * *` |student |view completed assignments |be assured I have completed an assignment
+
+|`* * *` |forgetful student |view overdue assignments |be assured I did not miss any deadline
+
+|`* * *` |forgetful student |get notification for upcoming assignments |never forget to complete my assignments
+
+|`* * *` |forgetful student |get notification for upcoming events |never forget to attend my events
+
+|`* *` |busy student |sort assignments based on priority |allocate sufficient time for each assignment
+
+|`* *` |busy student |filter assignments based on time period |view only assignments with deadlines within a certain time period
+
+|`* *` |busy student |filter events based on time period |view only events within a certain time period
+
+|`* *` |busy student |filter assignments based on tags |view only certain assignments when there are too many assignments to view
+
+|`* *` |busy student |filter events based on tags |view only certain events when there are too many events to view
+
+|`* *` |procrastinating student |add estimated time needed to complete an assignment |estimate when to start on an assignment to complete it on time
+
+|`* *` |procrastinating student |view contact details of person in-charge for a deadline extension |have enough time to complete my assignments despite my tight schedule
+
+|`* *` |busy student |view free time slots before a deadline |know how much time I have when I'm actually free to finish my tasks
+
+|`* *` |student |view common free time slots among my group mates |arrange a group meeting
+
+|`* *` |organised student |add multiple deadlines for an assignment |keep track of project progress
+
+|`* *` |NUS student |import timetable from NUSMods |avoid the trouble of keying in my lecture and tutorial schedules manually
+
+|`*` |student |be rewarded for completing an assignment |feel a sense of achievement
+
+|`*` |student |hide private events |minimize chance of someone else seeing them by accident
+|=======================================================================
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `TaskManager` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+
+=== Use case: Add assgnment
+
+*MSS*
+
+1. User requests to add in new assignment
+2. User supplies the detail of assignment as format specified by the command
```

Comment:
Recall on our previous tutorial on Use case is that the interaction should alternate between the user and system.
So in this case it would be
```
1. User blah blah
2. System blah blah
3. User blah blah
4. System blah blah
```